### DISCUSSION TITLE
One-Click "Not Your Channel" Report batch file

### DISCUSSION THREAD

**Discussion by Maddcow** [ThioJoe/YT-Spammer-Purge#693] [Show and tell]
I do not see a way to automate this 100% for those that are running this in "Not Your Channel" mode. To make a one-click solution, I updated my config file with the appropriate values and the channel that I want to scan. Then I created a batch file with the following commands to open YTSpammerPurge, send the "REPORT" input command, and then exit the programs. It's a work around but the process can now be scheduled to report for that Youtube Channel. 

@echo off
title YoutTube Spammer Purge
echo REPORT|YTSpammerPurge.2.14.3.exe

**Comment by DinhHuy2010** [comment]
what???

**Comment by LunaticBFF57** [comment]
What is this meant to be?

**Comment by Firecul** [comment]
> I do not see a way to automate this 100% for those that are running this in "Not Your Channel" mode. 
To make a one-click solution, I updated my config file with the appropriate values and the channel that I want to scan. Then I created a batch file with the following commands to open YTSpammerPurge, send the "REPORT" input command, and then exit the programs. It's a workaround but the process can now be scheduled to report for that Youtube Channel. 

```bat
@echo off
title YoutTube Spammer Purge
echo REPORT|YTSpammerPurge.2.14.3.exe

**Comment by Firecul** [comment]
I've never tried to automate this program yet but if that bat file works it could be handy I guess.
~~I'll test it soon~~

I guess I'm doing something wrong but it does report the comments automatically.
![image](https://user-images.githubusercontent.com/34950371/175395160-6dc0ab82-334f-4c69-a789-c9d93f7325f5.png)
